code/game/objects/items/weapons/clown_items.dm

```diff
@@ -172,33 +172,38 @@
 		return
 
 	if(spent)
-		user << "<span class='warning'>There's no glue left in the bottle.</span>"
+		to_chat(user,"<span class='warning'>There's no glue left in the bottle.</span>")
 		return
 
 	if(!istype(target)) //Can only apply to items!
-		user << "<span class='warning'>That would be such a waste of glue.</span>"
+		to_chat(user,"<span class='warning'>That would be such a waste of glue.</span>")
 		return
 	else
 		if(istype(target, /obj/item/stack)) //The whole cant_drop thing is EXTREMELY fucky with stacks and can be bypassed easily
-			user << "<span class='warning'>There's not enough glue in \the [src] to cover the whole [target]!</span>"
+			to_chat(user,"<span class='warning'>There's not enough glue in \the [src] to cover the whole [target]!</span>")
 			return
 
 		if(target.abstract) //Can't glue TK grabs, grabs, offhands!
 			return
 
-	user << "<span class='info'>You gently apply the whole [src] to \the [target].</span>"
+	to_chat(user,"<span class='info'>You gently apply the whole [src] to \the [target].</span>")
 	spent = 1
 	update_icon()
 	apply_glue(target)
 
 /obj/item/weapon/glue/proc/apply_glue(obj/item/target)
 	src = null
-
-	target.cant_drop++
+	if(istype(target, /obj/item/clothing))
```

Comment:
Can you make this use a proc instead and just override it.

Also use the ++ and -- method that cant_drop also uses. That way things that are already can_remove = 0 are preserved.